central/reports/scheduler/env.go

```diff
@@ -0,0 +1,22 @@
+package scheduler
```

Comment:
Please consider moving this code to some shared package in case branding logic will be reused in different places. The fact that it is now in `central/reports/scheduler` would require us to move it in the future, should we want to produce conditional branding somewhere else.
I think there were sibling JIRAs to ROX-9929 that may need this too. Also, there's Operator where customized branding could help.

Reply:
Moved it to pkg/branding/env.go

Reply:
Looks good to me.